nodewatcher/modules/platforms/openwrt/cgm/general.py

```diff
@@ -66,7 +66,16 @@ def usb(node, cfg):
         # Include base USB packages for devices supporting USB.
         # TODO: Perhaps this should be made device-specific so only the needed packages are installed.
         cfg.packages.update([
-            'kmod-usb2',
             'kmod-usb-ohci',
             'kmod-usb-uhci',
         ])
+        # Include packages to support USB3.0 if device support it.
+        if device.usb3:
```

Comment:
Where did you get this `device.usb3` from? You need to add support for this to device descriptors, otherwise this will not work.

And I also don't like having two flags (`usb` and `usb3`), it would be much better to have something like this in the device descriptors:
```python
# Device does not support USB (can be either of these).
usb = False
usb = None
# Device supports USB2.
usb = cgm_devices.USB_2
# Device supports USB3.
usb = cgm_devices.USB_3
```

Reply:
Ok, I like your idea.
Where are those flags defined?

Reply:
In `nodewatcher.core.generator.cgm.devices`.

Reply:
Finally looked into cgm.devices, and I don't think I understood your idea.
This way also we need 2 flags

Reply:
What do you mean two flags? You only need one `usb` attribute/flag in the device descriptor instead of the recommended two (`usb` and `usb3`). For example I do not want to have something like this in the device descriptor for a device, which supports USB3:
```python
class MyShinyDevice(cgm_devices.DeviceBase):
    """
    My shiny device v1 device descriptor.
    """

    identifier = 'my-shinyv1'
    name = "My Shiny (v1)"
    manufacturer = "Shiny Technologies"
    url = 'http://www.shiny-tech.com/'
    architecture = 'ar71xx'
    usb = True
    usb3 = True
    # ...
```

Instead I only want to have a single attribute to configure USB, the `usb` attribute. So in order to support different USB protocol versions, we need to change how the existing attribute works. My proposal is to change the values that the `usb` attribute can take. 

You need to do the following:
* Define two new constants in `devices.py`, `USB_2` and `USB_3` (e.g., with values `'usb-2'` and `'usb-3'`).
* Change all CGMs (I think this is actually just one, the one that you already changed in this PR, but you should check), which use the `usb` attribute to support the new constants.
* Change the device descriptor validation code to check that the `usb` attribute takes one of the supported values (either `None`, `False`, `USB_2` or `USB_3`).
* Change all existing device descriptors to use `USB_2` instead of `True`.